src/loaders/svg/tvgSvgSceneBuilder.cpp

```diff
@@ -264,6 +263,8 @@ static void _applyComposition(Paint* paint, const SvgNode* node, const Box& vBox
             for (uint32_t i = 0; i < compNode->child.count; ++i, ++child) {
                 if (_appendChildShape(*child, comp.get(), vBox, svgPath)) valid = true;
             }
+            comp->fill(255, 255, 255, 255);
```

Comment:
probably, we can call the fill()/opacity() in the valid condition
```
if (valid) {
   comp->fill();
   comp->opacity();
   paint->composite(move(comp), CompositeMethod::ClipPath);
}

Reply:
done